CMSIS/NN/Source/SoftmaxFunctions/arm_softmax_s8.c

```diff
@@ -21,8 +21,8 @@
  * Title:        arm_softmax_s8.c
  * Description:  S8 softmax function
  *
- * $Date:        09. October 2020
- * $Revision:    V.2.0.1
+ * $Date:        01. March 2021
+ * $Revision:    V.2.0.2
```

Comment:
This would be a 3.0.0 change as it is a change in the API..

Reply:
Ok. I suppose the same goes for arm_nnfunctions.h